app/src/main/java/fr/free/nrw/commons/theme/NavigationBaseActivity.java

```diff
@@ -187,14 +187,28 @@ public boolean onNavigationItemSelected(@NonNull final MenuItem item) {
                 return true;
             case R.id.action_feedback:
                 drawerLayout.closeDrawer(navigationView);
+
```

Comment:
@madhurgupta10 Can you extract `CommonsLogSender:getExtraInfo` to `Utils` and use the same method at both the places. :)

Feel free to add any extra meta info that is not added in `getExtraInfo`. 

Having the same method at both places would make it easier to app/remove any meta data being sent along with the logs/feedback mail.

Reply:
Sure, but the CommonsLogSender.getExtraInfo() does not have localization for different languages. Should I replace the hardcoded strings with **res.string**?

Reply:
I am hesitant in localising the strings at any of these two places. Let's discuss this on the issue thread